pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -37,10 +37,10 @@ func parseControllerCerts(ctx *zedagentContext, contents []byte) {
 	if bytes.Equal(newHash, certHash) {
 		return
 	}
-	log.Infof("parseControllerCerts: Applying updated config\n"+
-		"Last Sha: % x\n"+
-		"New  Sha: % x\n"+
-		"Num of cfgCert: %d\n",
+	log.Infof("parseControllerCerts: Applying updated config"+
+		"Last Sha: % x"+
+		"New  Sha: % x"+
+		"Num of cfgCert: %d",
```

Comment:
I think this was intended to be human readable with line breaks. But perhaps that doesn't make sense with json.
But in any case, you need to replace the intermediate \n with a space or something - right now you are concantating  words and sha values.

Reply:
Resolved